tests/css_case/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -292,17 +300,17 @@ a[href="KeepAttrValue"]:hover::first-letter,
 }
 
 @keyframes KeepAnimationName {
-  from {
+  FROM {
     prop: val;
   }
 
   #{$KeepInterpolationVar},
-  #{$Keep + 15px},
+  #{$Keep + 15PX},
```

Comment:
Oops, I didn't see this one when looking through the diff before. Need to lowercase that "PX".